fitbenchmarking/fitbenchmarking
# 770 fix bumps parameter ordering

#### Description of Work

Fixes #770

#### Testing Instructions

1. Run Fitbenchmarking with SASView problems and bumps fitting software to check results are now correct

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
